--- premake5_rive_plugin.lua	2023-11-08 21:16:37.466026217 +0100
+++ premake5_rive_plugin.lua	2023-11-08 21:36:12.735951780 +0100
@@ -101,7 +101,8 @@
         linkoptions {
             '-Oz',
             '-g0',
-            '-flto'
+            '-flto',
+            '--build-id=None'
         }
     end
     filter {'system:macosx', 'options:arch=arm64'}
@@ -452,7 +453,8 @@
         linkoptions {
             '-Oz',
             '-g0',
-            '-flto'
+            '-flto',
+            '--build-id=None'
         }
     end
 
